From 776ce2aa34061d92c404c5964afb6bfab65c1382 Mon Sep 17 00:00:00 2001
From: Rahul T R <r-ravikumar@ti.com>
Date: Tue, 15 Nov 2022 16:35:44 +0530
Subject: [PATCH 3/3] drm/bridge: cdns-dsi: Fix issue with phy init

Phy is not being initialized after suspend resume. Fix this by setting
phy_initialized flag to false in suspend callback

Upstream Link: https://lore.kernel.org/all/20221115083918.23192-1-r-ravikumar@ti.com/

Signed-off-by: Rahul T R <r-ravikumar@ti.com>
---
 drivers/gpu/drm/bridge/cadence/cdns-dsi-core.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/drivers/gpu/drm/bridge/cadence/cdns-dsi-core.c b/drivers/gpu/drm/bridge/cadence/cdns-dsi-core.c
index cce51ee51dfd..aab961daf7c7 100644
--- a/drivers/gpu/drm/bridge/cadence/cdns-dsi-core.c
+++ b/drivers/gpu/drm/bridge/cadence/cdns-dsi-core.c
@@ -754,6 +754,7 @@ static int __maybe_unused cdns_dsi_suspend(struct device *dev)
 	clk_disable_unprepare(dsi->dsi_p_clk);
 	reset_control_assert(dsi->dsi_p_rst);
 	dsi->link_initialized = false;
+	dsi->phy_initialized = false;
 	return 0;
 }
 
-- 
2.30.2

